ci/rebuild-docker-images.py

```diff
@@ -485,6 +488,98 @@ def playwright_build(distro, arch):
 
 
 
+def appium_build(distro, arch):
+    tag = f"{registry_base}{distro[0]}-{distro[1]}"
+    build_tag(tag, arch, f"""
+FROM openjdk:18-ea-11-jdk-slim-bullseye
+
+ENV HOME=/root \
+    LANG=en_US.UTF-8 \
+    DEBIAN_FRONTEN=noninteractive \
+    LANGUAGE=en_US.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    DISPLAY=:0.0 \
+    DISPLAY_WIDTH=1920 \
+    DISPLAY_HEIGHT=900
+
+
+WORKDIR /
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update
+RUN apt install -y ca-certificates curl git cpu-checker supervisor vim bash wget unzip xvfb x11vnc fluxbox xterm novnc net-tools htop libpulse-dev libnss3 libxcursor1 libasound2 libqt5gui5 libc++-dev libxcb-cursor0 htop tree tar gzip
+
+
+ARG ARCH="x86_64"
+ARG TARGET="google_apis_playstore"
+ARG API_LEVEL="{distro[1]}"
+ARG BUILD_TOOLS="{distro[1]}.0.0"
+ARG ANDROID_API_LEVEL="android-$API_LEVEL"
+ARG ANDROID_APIS="$TARGET;$ARCH"
+ARG EMULATOR_PACKAGE="system-images;$ANDROID_API_LEVEL;$ANDROID_APIS"
+ARG PLATFORM_VERSION="platforms;$ANDROID_API_LEVEL"
+ARG BUILD_TOOL="build-tools;$BUILD_TOOLS"
+ARG ANDROID_CMD="commandlinetools-linux-11076708_latest.zip"
+ARG ANDROID_SDK_PACKAGES="$EMULATOR_PACKAGE $PLATFORM_VERSION $BUILD_TOOL platform-tools"
+
+ENV ANDROID_SDK_ROOT=/opt/android
+ENV PATH "$PATH:$ANDROID_SDK_ROOT/cmdline-tools/tools:$ANDROID_SDK_ROOT/cmdline-tools/tools/bin:$ANDROID_SDK_ROOT/emulator:$ANDROID_SDK_ROOT/tools/bin:$ANDROID_SDK_ROOT/platform-tools:$ANDROID_SDK_ROOT/build-tools/$BUILD_TOOLS"
+ENV DOCKER="true"
+
+#============================================
+# Install required Android CMD-line tools
+#============================================
+RUN wget https://dl.google.com/android/repository/$ANDROID_CMD -P /tmp && \
+              unzip -d $ANDROID_SDK_ROOT /tmp/$ANDROID_CMD && \
```

Comment:
You should add a `&& rm -f /tmp/$ANDROID_CMD` after unzipping (so that the temporary zip file doesn't have to get put into the image.)